### PR TITLE
Minor fixes

### PR DESCRIPTION
**Pull request by xavierlepretre**
This PR beautifies spacing.

### Change scope

The changes in this Pull Request include (please tick all that apply):

* [ ] Small language/grammar fixes
* [ ] Small content fixes 
* [ ] Addition of new content
* [ ] Sample code/command updates
* [ ] Platform fixes
* [ ] Other

### CODE REVIEW

**Comment by xavierlepretre** [comment]
@coldice I think this one can go straight in.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # CODEOWNERS: https://help.github.com/articles/about-codeowners/
 
-# Docs 
-*.md @citmc @cmwaters @nooomski @xavierlepretre @coldice 
+# Docs
+*.md @citmc @cmwaters @nooomski @xavierlepretre @coldice
```
Comment (coldice) [inline .github/CODEOWNERS:4]: GitHub doesn't like @cmwaters anymore - I think the account doesn't exist anymore, can you remove it?

Comment (xavierlepretre) [reply .github/CODEOWNERS:4]: More like the account is no longer an owner.
That's quite the verification by Github!

Comment (xavierlepretre) [reply .github/CODEOWNERS]: Done.

File: hands-on-exercise/5-ibc-adv/5-ibc-app-packets.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 title: "Adding Packet and Acknowledgement Data"
 order: 6
-description: 
-tags: 
+description:
```
Comment (coldice) [inline hands-on-exercise/5-ibc-adv/5-ibc-app-packets.md:4]: shouldn't add empty fields - let's set or remove

Comment (coldice) [reply hands-on-exercise/5-ibc-adv/5-ibc-app-packets.md:4]: Ok I expect this is on prupose as we have a few more sections with that, to overwrite the default?

**Review by coldice — APPROVED** [review]

**Pull request merged**